pkgs/development/libraries/gnutls/default.nix

```diff
@@ -163,7 +163,7 @@ stdenv.mkDerivation rec {
     ]
     ++ lib.optionals doCheck [
       which
-      nettools
```

Comment:
@maralorn it looks like liquid-fixpoint is already marked as broken in https://github.com/zimbatm/nixpkgs/blob/4442a92514a1ae88e81d5969f4b43298735f0404/pkgs/development/haskell-modules/configuration-hackage2nix/broken.yaml#L3646

Are those still being generated and pushed to nixpkgs?

Reply:
Ideally we can remove the package if it's broken, so I don't have to worry about cabal2nix backward/future-compatibility.

Reply:
Nothing prevents you from bumping cabal2nix in this PR in principle.

Reply:
Done. Can we consider this point as "solved"?

Reply:
Done